UI delete account

Chanpech opened the pull request:
UI still needs some clean-up on the responsive menu and the code itself but I am short on time so I am going to PR what I have right now. There is no error and the side nav should be able to take you to this account page.

Chanpech committed:
Created the account page and dialog feature for delete account. Will need assistance for cleaning up some of the codes

Chanpech committed:
UI delete_account small change

Made a couple of small changes to the email ui.

Chanpech committed:
Removed the import because it was not used.

I was going to clean up some of my code but I end up running short on time. So I am pushing what I have

tjzetty commented on apps/frontend/src/components/App.tsx:
```diff
@@ -58,8 +77,41 @@ export default function App() {
+                {settings.map((setting) => (
+                  <MenuItem
+                    key={setting}
+                    onClick={() => handleCloseUserMenu(setting)}
```
There should be two onClick events. If you click account it should link to the account page, if you click logout it should do undefined until we wire up the backend.

Chanpech replied:
I just corrected it. Now everything should be all set.

tjzetty reviewed (changes requested):
Make account button link to the account page after clicking the user profile button (top right of homepage).

Chanpech committed:
Removed the responsive menu and other small changes

I removed the responsive menu which used to display the account and logout when profile is clicked. Now when profile is clicked it takes user to the account page. I also separated the username to firstname and lastname

tjzetty committed:
Merge branch 'main' into ui_delete_account
